src/iop/toneequal.c

```diff
@@ -2374,14 +2374,19 @@ void gui_post_expose(struct dt_iop_module_t *self, cairo_t *cr, int32_t width, i
   // set custom cursor dimensions
   const double outer_radius = 16.;
   const double inner_radius = outer_radius / 2.0;
-  const double setting_scale = 2. * outer_radius / zoom_scale;
+  //const double setting_scale = 2. * outer_radius / zoom_scale;
```

Comment:
Please remove commented out code. Without a comment this will not help anyway.

Reply:
oops, sorry.